ocaml/runtime/caml/exec.h

```diff
@@ -60,7 +60,7 @@ struct exec_trailer {
 
 /* Magic number for this release */
 
-#define EXEC_MAGIC "Caml1999X033"
+#define EXEC_MAGIC "Caml1999X520"
```

Comment:
I wonder whether we should use a different
magic number since this runtime has additional
opcodes.

Reply:
Agreed - have made a note elsewhere to deal with this.